interface/handlers/champions.go

```diff
@@ -0,0 +1,128 @@
+package handlers
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	"github.com/wizelineacademy/golang-bootcamp-2020/domain/models"
+	"github.com/wizelineacademy/golang-bootcamp-2020/domain/repositories"
+	"github.com/wizelineacademy/golang-bootcamp-2020/helpers"
+)
+
+// ChampionHandler defines the Champion Handler properties
+type ChampionHandler struct {
+	infoLog   *log.Logger
+	errorLog  *log.Logger
+	champRepo repositories.ChampionRepository
+}
+
+// NewChampionHandler returns a ChampModel struct with a logger.
+func NewChampionHandler(infoLog, errorLog *log.Logger, cr repositories.ChampionRepository) *ChampionHandler {
+	return &ChampionHandler{infoLog, errorLog, cr}
+}
+
+// GetChamp returns a single Champion by id
+func (ch *ChampionHandler) GetChamp(w http.ResponseWriter, r *http.Request) {
+	// Get the Gorilla mux vars (path params)
+	vars := mux.Vars(r)
+	id, err := strconv.Atoi(vars["id"])
+	if err != nil || id < 1 {
+		http.Error(w, http.StatusText(http.StatusNotFound), http.StatusNotFound)
+		return
+	}
+
+	// Query the database to get a single Champions
+	c, err := ch.champRepo.GetSingle(id)
+	if err != nil {
+		if errors.Is(err, repositories.ErrNoRecord) {
+			http.Error(w, http.StatusText(http.StatusNotFound), http.StatusNotFound)
+		} else {
+			helpers.ServerError(w, ch.errorLog, err)
+		}
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	// Marshal the Champion struct to JSON
+	err = c.ToJSON(w)
+	if err != nil {
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		ch.errorLog.Printf("message: error while parsing the champions data, handler: champions, method: GET, id: %v, err: %v\n", id, err)
+	}
+}
+
+// GetChamps returns multiple Champions. It defaults to 10 if a limit isn't specified or if the limit is < 1.
+func (ch *ChampionHandler) GetChamps(w http.ResponseWriter, r *http.Request) {
+
+	const (
+		defaultLimit = 10
+	)
```

Comment:
Single const definition shouldn't be a const block.
```suggestion
	const defaultLimit = 10
```